### PR TITLE
Story/claudia 5159

### PR DESCRIPTION
**Pull request by hmunfru**
@jesuspg @jframos 
Testbed locally agains FILAB 
https://jira.tid.es/browse/CLAUDIA-5160

### CODE REVIEW

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/claudia/impl/ClaudiaClientOpenStackImpl.java [hunk 1]
```diff
@@ -220,12 +224,26 @@ public String getUserData(ClaudiaData claudiaData, TierInstance tierInstance) {
...
                 .replace("{validation_key}", chefValidationKey).replace("{networks}", writeInterfaces(tierInstance))
-                .replace("{if_up}", generateIfUp(tierInstance)).replace("{puppet_master}", puppetHostname);
+                .replace("{if_up}", generateIfUp(tierInstance)).replace("{puppet_master}", puppetHostname)
+                .replace("{support_key}", key);
         log.debug("payload " + file);
         return file;
 
     }
 
+    public String getSupportKey (String region)  {
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/claudia/impl/ClaudiaClientOpenStackImpl.java]: Provide description of the new method.

**Comment by jesuspg** [comment]
LGTM

**Comment by jframos** [comment]
LGTM

**Comment by jframos** [comment]
This PR is not completed. SystemPropertiesProvider.properties should be updated.

**Pull request merged**